crates/bevy_render/src/color/mod.rs

```diff
@@ -1732,6 +1732,71 @@ impl encase::private::CreateFrom for Color {
 
 impl encase::ShaderSize for Color {}
 
+impl Lerp for Color {
```

Comment:
Does this use the special color mixing that take brightness into account or the naive color gradient algorithm?